File: enrml.jl
include("setup.jl")

Ne = 25
γ = 10
i_max = 4

θs, fs, us, ss, λs, inds = run_lm_enrml(f, g, p, L, γ, i_max, n_blocks, Ne, localisation=true)

@save "results/enrml_$(Ne).jld2" θs fs us ss λs inds

File: enrml.py
import ensemble_methods as em

from setup import *

Ne = 100

prob = em.EnRMLProblem(f, g, prior, likelihood, Nf, Ne, i_max=30, loc_type="cycle")
prob.run()
prob.save_results(fname=f"data/enrml_{Ne}_loc_cycle_max_30_tsvd_99")